openedx/core/djangoapps/theming/storage.py

```diff
@@ -219,6 +226,88 @@ def converter(matchobj):
 
         return converter
 
+    def _url_converter__gte_111(self, name, hashed_files, template=None):
+        """
+        This is an override of url_converter from CachedFilesMixin.
```

Comment:
I realize the method name basically says this but please add a note in the comment about this being in place for Django 1.11+